The `dart pub uploader` command will be removed in Dart 2.17 (#3882)
dart-lang/site-www: src/tools/pub/cmd/pub-uploader.md

description: Use dart pub uploader to add or remove uploaders for your Dart package on the pub.dev site.
---

{{site.alert.warning}}
  The `dart pub uploader` command is _deprecated_ and will be removed in Dart

>   2.17. Package owners can manage uploaders using the admin page on pub.dev:

Reviewer: Should we add something like:
```suggestion
  2.17. And will stop working for earlier versions too. Package owners can manage uploaders using the admin page on pub.dev:
```

Author: I'm not sure there is any reason to stop older versions from using the API.
But I don't oppose to it, since it simplifies code on pub.dev, and that is good :D

Author: Also filed https://github.com/dart-lang/pub-dev/issues/5566

>   The `dart pub uploader` command is _deprecated_ and will be removed in Dart
>   2.17. And will stop working for earlier versions too. Package owners can manage uploaders using the admin page on pub.dev:
>   `https://pub.dev/packages/<package>/admin`.

Reviewer 2: ```suggestion
  The `dart pub uploader` command is _deprecated_ and will be
   removed in Dart 2.17. It won't work for earlier versions, either.
  Package owners can manage uploaders using the admin page on pub.dev:
  `https://pub.dev/packages/<package>/admin`.
```

bob@example.com has been removed as an uploader for this package.
```

{{site.alert.tip}}

>   Instead of specifying uploaders for each package you publish,
>   consider using a
>   [verified publisher](/tools/pub/verified-publishers).

Reviewer 2: ```suggestion
  Instead of specifying uploaders for each package that you publish,
  consider using a
  [verified publisher](/tools/pub/verified-publishers).
```
